cli_tools/common/utils/daisyutils/daisy_worker.go

```diff
@@ -62,19 +62,19 @@ func createResourceLabelerIfMissing(env EnvironmentSettings, modifiers []Workflo
 }
 
 type defaultDaisyWorker struct {
-	wf        *daisy.Workflow
-	logger    logging.Logger
-	env       EnvironmentSettings
-	modifiers []WorkflowModifier
+	wf     *daisy.Workflow
+	logger logging.Logger
+	env    EnvironmentSettings
+	hooks  []WorkflowHook
 }
 
 // Run runs the daisy workflow with the supplied vars.
 func (w *defaultDaisyWorker) Run(vars map[string]string) error {
-	if err := (&ApplyAndValidateVars{w.env, vars}).Modify(w.wf); err != nil {
+	if err := (&ApplyAndValidateVars{w.env, vars}).PreRunHook(w.wf); err != nil {
 		return err
 	}
-	for _, t := range w.modifiers {
-		if err := t.Modify(w.wf); err != nil {
+	for _, t := range w.hooks {
```

Comment:
What does t stand for here? Not sure what it even stood for when modifiers were used tbh.